api/v1alpha1/promise_types.go

```diff
@@ -347,6 +415,15 @@ func (p *Promise) GetWorkloadGroupScheduling() []WorkloadGroupScheduling {
 	return workloadGroupScheduling
 }
 
+// TotalWorkflows returns the total number of workflows defined on the Promise.
+func (p *Promise) TotalWorkflows() int64 {
```

Comment:
How this is calculated is off. We only care about the promise configure workflow.